src/app/pages/MediaArticlePage/PagePromoSections/LastestMediaSection/LatestMediaItem/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import isEmpty from 'ramda/src/isEmpty';
+import Promo from '#components/OptimoPromos';
+import { LatestMediaItemProp } from '../LatestMediaTypes';
+import LatestMediaIndicator from '../LatestMediaIndicator';
+import {
+  StyledPromoTitle,
+  StyledTimestamp,
+  styles,
+  StyledPromo,
+} from './index.styles';
+
+const LatestMediaItem = ({
+  item,
+  ariaLabelledBy,
+  eventTrackingData,
+  ref,
+}: LatestMediaItemProp) => {
+  if (!item || isEmpty(item)) return null;
+
+  const timestamp = item.firstPublished;
+
+  const src = item.imageUrl.replace('{width}', '240');
+
+  return (
+    <div ref={ref} css={styles.promoWrapper}>
+      <StyledPromo
```

Comment:
Similar to the other comment, I think we can import the `Promo` component directly here and use the `css` prop to pass the styles, rather than styling it in the CSS file and then importing as a Styled component.